tools/mesh-generators/requirements.txt

```diff
@@ -1,2 +1,5 @@
 gmsh>=4.10.0
 meshio>=5.3.4
+numpy
+scipy
+vtk
```

Comment:
It's not a complete no-go, but would it be possible to rely on the vtk version shipped with the cpp version? Installing this on my system will lead to conflicts when using aste later on. The generators are not in the main workflow loop, though.

Reply:
I saw your discussion on this lately. I will remove vtk from here.

Reply:
Note that the generator did not work for me without installing the python vtk package explicitly.

Reply:
Shall I rely on `meshio` it is easy to convert but I assume if one uses the mesh generator they also want to use at least `precice-aste-evaluate` which requires vtk package

Reply:
For the evaluate package, the cpp bundled (bindings?) vtk version is sufficient, so yes, would be nice.